features/support/capybara_selectors/paragraph_link_named.rb

```diff
@@ -0,0 +1,6 @@
+Capybara.add_selector(:paragraph_link_named) do
```

Comment:
This selector seems to be nothing to do with a link ?  it is more like 'paragraph_containing' or something like that ?